releases/release-4.0.6.md

```diff
@@ -0,0 +1,130 @@
+    - Fix a runtime error in `TRUNCATE` [#19445](https://github.com/pingcap/tidb/pull/19445)
+    - Add `TiDB-Runtime` dashboard [#19396](https://github.com/pingcap/tidb/pull/19396)
+    - Ddl: exit add index on generated column with `case-when` expression parse error [#19395](https://github.com/pingcap/tidb/pull/19395)
+    - Fix issue 19371 - PREPARE statement FROM @Var (where Var contains uppercase characters fail) [#19378](https://github.com/pingcap/tidb/pull/19378)
+    - Improve compatibility for ALTER TABLE algorithms [#19364](https://github.com/pingcap/tidb/pull/19364)
+    - Fix a schema charset modification panic bug in an uppercase schema. [#19302](https://github.com/pingcap/tidb/pull/19302)
+    - Planner: encode insert/delete/update plan in slow log plan field. [#19269](https://github.com/pingcap/tidb/pull/19269)
+    - Fix inconsistence of Plan between information_schema.statements_summary and explain, which contains tikv/tiflash info. [#19159](https://github.com/pingcap/tidb/pull/19159)
+
++ TiKV
+
+    - Optimize QPS drop when doing UnsafeDestroyRange [#8627](https://github.com/tikv/tikv/pull/8627)
+    - Fix the estimation error for a non-index column with collation enabled. [#8620](https://github.com/tikv/tikv/pull/8620)
+    - Error-code: support to generate metafile [#8619](https://github.com/tikv/tikv/pull/8619)
+    - Add perf statistics for cf scan detail [#8618](https://github.com/tikv/tikv/pull/8618)
+    - Fix the deadlock between the PD client thread and other threads calling PD sync requests. [#8612](https://github.com/tikv/tikv/pull/8612)
+    - Fix a panic issue if a TiKV runs very slow during conf change. [#8497](https://github.com/tikv/tikv/pull/8497)
+    - Add perf context panel for grafana template [#8467](https://github.com/tikv/tikv/pull/8467)
+    - Update jemalloc to 5.2.1 [#8463](https://github.com/tikv/tikv/pull/8463)
+    - Fix the issue that green GC may miss locks during transferring regions. [#8460](https://github.com/tikv/tikv/pull/8460)
+    - Support float CPU quota in cgroup. [#8427](https://github.com/tikv/tikv/pull/8427)
+
++ PD
+
+    - Dashboard: update to v2020.09.08.1 [#2928](https://github.com/pingcap/pd/pull/2928)
+    - Add `initial-cluster-token` configuration to avoid different clusters communicate during bootstrap [#2922](https://github.com/pingcap/pd/pull/2922)
+    - Change back to the original way to control the low space threshold [#2875](https://github.com/pingcap/pd/pull/2875)
+    - Fix the unit of store limit rate when the mode is auto [#2826](https://github.com/pingcap/pd/pull/2826)
+    - Add service gc safepoit commands [#2813](https://github.com/pingcap/pd/pull/2813)
+    - Fix the issue that no filtered unhealthy store in `summaryStoresLoad` [#2805](https://github.com/pingcap/pd/pull/2805)
+    - Fix a bug that pd will not pass the vet check in go1.15. [#2753](https://github.com/pingcap/pd/pull/2753)
```

Comment:
+ PD

    - Add `initial-cluster-token` configuration to avoid different clusters communicate during bootstrap [#2922](https://github.com/pingcap/pd/pull/2922)
    - Fix the unit of store limit rate when the mode is auto [#2826](https://github.com/pingcap/pd/pull/2826)
- Fix the issue that some scheduler persist config without solving error [#2818](https://github.com/tikv/pd/pull/2818)
    - metrics: add more metrics for region and store heartbeat [#2891](https://github.com/tikv/pd/pull/2891)
    - fix empty http response in scheduler [#2871](https://github.com/tikv/pd/pull/2871) [#2874](https://github.com/tikv/pd/pull/2874)

Reply:
Updated.